client/state/jetpack-remote-install/reducer.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { createReducer, combineReducers, keyedReducer } from 'state/utils';
+import {
+	JETPACK_REMOTE_INSTALL,
+	JETPACK_REMOTE_INSTALL_FAILURE,
+	JETPACK_REMOTE_INSTALL_SUCCESS,
+} from 'state/action-types';
+
+export const isComplete = keyedReducer(
+	'url',
+	createReducer( false, {
```

Comment:
I've observed a tendency to discourage `createReducer` as an alternative to building simple reducers with a `switch` as it adds some overhead and offers limited benefit.

I'm fine with this either way.